src/test/java/mate/academy/springbootintro/service/BookServiceTest.java

```diff
@@ -57,22 +72,19 @@ public void createBook_ValidRequestDto_ReturnsBookDto() {
         // Given
         Category category1 = new Category();
         category1.setId(CATEGORY_ID_1);
-        category1.setDescription("fantasy");
-        category1.setDescription("dragons");
+        category1.setName(CATEGORY_NAME_1);
+        category1.setDescription(CATEGORY_DESCRIPTION_1);

```

Comment:
You should create and use something similar to `createBookDto` you created in BookControllerTest